app/helpers/questions_helper.rb

```diff
@@ -18,13 +18,15 @@ def next_question(current_question)
     if current_question_index.nil?
       questions_to_ask.first
     elsif current_question_index == (questions_to_ask.length - 1)
-      RESULTS_PAGE
+      FINAL_QUESTION
     else
       questions_to_ask[current_question_index + 1]
     end
   end
 
   def previous_question(current_question)
+    return questions_to_ask.last if current_question == FINAL_QUESTION
+
     current_question_index = questions_to_ask.index(current_question)
     if current_question_index.zero?
       FILTER_QUESTION
```

Comment:
So, I guess this could be:
```
return FILTER_QUESTION if current_question == questions_to_ask.first
```
as well?

Reply:
I feel like I'm understanding this module better now.

Reply:
Added an additional commit to change the logic there.  Making this explicit does feel like a better thing to do.

Reply:
Thanks! It looks good ✅

Reply:
Just seen this: `questions_to_ask[current_question_index + -1]` 🙈  Was that just meant to be `current_question_index - 1`